backend/sh/copy_file_from_image.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```

Comment:
is it really the better approach to add another backend file duplicating a lot of the already existing one instead of making the backend script handling both?
my answer would be 'no' given the amount of shared code.

Reply:
@HarryWaschkeit 
my answer is yes with respect to the current way we call the script in docker from rust (-;

I get your point and I was thinking about that as well. The amount of effort for refactoring the script and rust code was to high for me. You would have to specify a complete new cli for copying files in two directions.  currently there is a 1:1 mapping between omnect-cli cmd and command in backend which is represented by a dedicated file. 

If you make a diff of both files there are not much lines of code the same, except the first part for the cli checks.